CSS/style.css

```diff
@@ -1502,9 +1503,9 @@ textarea {
     margin-bottom: -500px;
   }
 
-   /* ============================Start of Desktop-Details Popup Window======================= */
+  /* ============================Start of Desktop-Details Popup Window======================= */
 
-   .modal-content {
+  .modal-content {
```

Comment:
- [ ] Good job adding the popup 👍 .  I can see that your modal overlap the viewport for the mobile view. Kindly fix it

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181949694-efedb298-35dd-443c-9801-e8e1b69f33f6.png)  | ![image](https://user-images.githubusercontent.com/63473584/181951107-463700a1-1c1a-4d7a-837f-5f032093027a.png)  |